types/seamless-immutable/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Seamless-immutable 7.1
+// Type definitions for Seamless-immutable 7.2
```

Comment:
Seamless Immutable is still on version 7.1.x, best to leave this as it was (@latest = 7.1.4)